nodes/pods/nodes-pods-secrets.adoc

```diff
@@ -21,6 +21,20 @@ include::modules/nodes-pods-secrets-creating.adoc[leveloffset=+1]
 
 include::modules/nodes-pods-secrets-creating-opaque.adoc[leveloffset=+2]
 
+include::modules/nodes-pods-secrets-creating-sa.adoc[leveloffset=+2]
+
+.Additional references
```

Comment:
This requires a [role="_additional-resources"] in the line before the title and should be titled "Additional resources"

Reply:
I do that all the time!